src/main/java/org/openstreetmap/atlas/geography/geojson/GeoJson.java

```diff
@@ -29,5 +29,17 @@
 {
     JsonObject asGeoJson();
 
+    /**
+     * Return a {@link JsonObject} representation that uses linestrings instead of polygons where
+     * applicable. This may be useful for certain object types, since many visualization softwares
+     * have an easier time showing linestrings.
+     *
+     * @return the {@link JsonObject} with any polygons represented instead as linestrings
+     */
+    default JsonObject asGeoJsonLineString()
+    {
+        throw new UnsupportedOperationException();
```

Comment:
Instead of having a default method with a throws exception here, I would just have it as a declared method

Reply:
Oh shoot. I actually added this but then decided not to use it and forgot to remove it. Do you think it is something users might want in the future? If not, I will just remove it from this PR.